lib/apiScenario/apiScenarioLoader.ts

```diff
@@ -427,7 +427,66 @@ export class ApiScenarioLoader implements Loader<ScenarioDefinition> {
       }
     };
 
-    if ("operationId" in rawStep) {
+    if ("exampleFile" in rawStep) {
+      // load example step
+      step.exampleFile = rawStep.exampleFile;
+
+      const exampleFilePath = pathJoin(pathDirName(ctx.scenarioDef._filePath), step.exampleFile!);
+
+      // Load example file
+      const fileContent = await this.fileLoader.load(exampleFilePath);
+      const exampleFileContent = JSON.parse(fileContent) as SwaggerExample;
+
+      let operation: Operation | undefined;
+
+      // Load Operation
+      if (exampleFileContent.operationId) {
+        step.operationId = exampleFileContent.operationId;
+
+        operation = this.operationsMap.get(step.operationId);
+        if (operation === undefined) {
+          throw new Error(`Operation not found for ${step.operationId} in step ${step.step}`);
+        }
+        if (this.opts.includeOperation) {
+          step.operation = operation;
+        }
+      } else {
+        const opMap = this.exampleToOperation.get(exampleFilePath);
+        if (opMap === undefined) {
+          throw new Error(`Example file is not referenced by any operation: ${step.exampleFile}`);
```

Comment:
we should use `rawStep.operationId` first, and then try to find operation from exampleFile